packages/material-ui-lab/src/SelectableGroup/SelectableGroup.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as utils from './utils';
+import SelectableGroupContext from './SelectableGroupContext';
+
+function SelectableGroup(props) {
+  const [selected, setSelected] = React.useState(null);
+  const { additional, children, exclusive, value: valueProp } = props;
+
+  const createReducer = handler => {
+    return (event, value) => {
+      setSelected(prevSelected => {
+        const newSelected = handler(prevSelected, value, exclusive);
+
+        // Only call onChange if it exists and state has changed (Object.is mimics React)
+        if (props.onChange && !Object.is(newSelected, prevSelected)) {
```

Comment:
Object.is is not supported in IE 11, why not using `===`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is#Polyfill

Reply:
Because it would be true for +-0 and false for NaN.

I wish react would use some dependency instead of polyfilling it themselves. Would make it essentially a free dependency for every react library.